cpp/include/cudf/lists/contains.hpp

```diff
@@ -163,7 +163,7 @@ std::unique_ptr<column> index_of(
  * @return std::unique_ptr<column> INT32 column of `n` rows with the location of the `search_key`
  *
  * @throw cudf::logic_error If `search_keys` does not match `lists` in its number of rows
- * @throw cudf::logic_error If `search_keys` type does not match the element type in `lists`
+ * @throw cudf::dtype_error If `search_keys` type does not match the element type in `lists`
```

Comment:
Please also remove the line below: it is no longer relevant.

Reply:
In addition, there is one more overload of `index_of` in this file that needs to be updated.

Reply:
Is the line below no longer relevant because of changes in this PR, or because prior work converted this function to use the experimental comparators?

Reply:
I updated the other overload as you suggested, thanks for catching that.